crates/cli/src/subcommands/identity.rs

```diff
@@ -454,6 +445,11 @@ async fn exec_import(mut config: Config, args: &ArgMatches) -> Result<(), anyhow
 
     //optional
     let nickname = args.get_one::<String>("name").cloned();
+    config.can_set_name(nickname.as_deref())?;
+
+    if config.identity_configs().iter().any(|ic| ic.identity == identity) {
+        return Err(anyhow::anyhow!("Identity already exists in config"));
+    };
```

Comment:
I've also added a check here to prevent adding the same identity multiple times

Reply:
(factored out `config.identity_exists` so it's somewhat self-describing)